src/System.Runtime.Loader/tests/ContextualReflectionDependency/Configurations.props

```diff
@@ -0,0 +1,8 @@
+﻿<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
This depends on netcoreapp-specific APIs. It should not be building for netstandard.